apiary-metastore-listener/README.md

```diff
@@ -105,8 +107,9 @@ The following shows an example JSON message representing an "ALTER_PARTITION" ev
       "eventType": "ALTER_PARTITION",
       "dbName": "some_db",
       "tableName": "some_table",
-      "partition": ["col_1","col_2"],
-      "oldPartition": ["col_1", "col_2", "col_3"]
+      "partitionKeys": ["col_1","col_2"],
+      "partitionValues": ["val_1", "val_2", "val_3"],
+      "oldPartitionValues": ["val_4", "val_5", "val_6"]
```

Comment:
Same here. To ensure it matches up with what the code does I would recommend generating these examples from the unit tests.